### PR TITLE
set default avatar in user , remove double assets

### PR DESCRIPTION
**Pull request by chmette**
hi @AdrianValdes ,

i cleaned up double assets and set the default avatar for the user

### CODE REVIEW

File: src/models/User.js [hunk 1]
```diff
@@ -9,7 +9,11 @@ const UserSchema = new Schema(
     nickname: { type: String, required: false },
     email: { type: String, required: true, unique: true },
     password: { type: String, required: true },
-    avatar: { type: String, required: true, default: 'yxz' },
+    avatar: {
+      type: String,
+      required: false,
+      default: './statics/avatar/01.png',
+    },
```
Comment (AdrianValdes) [inline src/models/User.js]: are we supposed to see this image by default in the front-end? I am just getting text.

Comment (chmette) [reply src/models/User.js]: we will after i insert the statics path

**Comment by chmette** [comment]
working with a fallback fill url now. 
i will implement the seed.js next and then take that matter again. do you agree?

File: src/models/Record.js [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
+    toJSON: { virtuals: true },
   }
 );
 
 const Record = model('Record', RecordSchema);
 
+// RecordSchema.virtual('url').get(() =>
+//   path.join(__dirname, this.avatar)
+// );
+
```
Comment (AdrianValdes) [inline src/models/Record.js]: Can we take this out?

File: src/models/Record.js [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
+      required: true,
+    },
     title: { type: String, required: true },
     artist: { type: String, required: true },
     year: { type: Number, required: true },
   },
   {
     versionKey: false,
     timestamps: true,
+    toJSON: { virtuals: true },
```
Comment (AdrianValdes) [inline src/models/Record.js]: Do we need this now?

File: src/models/User.js [hunk 4]
```diff
@@ -9,11 +9,16 @@ const UserSchema = new Schema(
...
+    avatar: {
+      type: String,
+      required: false,
+      default: 'http:localhost:5000/statics/avatar/01.png',
+    },
   },
   {
     versionKey: false,
     timestamps: true,
+    toJSON: { virtuals: true },
```
Comment (AdrianValdes) [inline src/models/User.js]: Do we need this?

File: server.js [hunk 5]
```diff
@@ -49,7 +51,7 @@ app.get('/', (req, res) => {
 });
 
 // Errorhandler
-app.use((err, req, res) => {
+app.use((err, req, res, next) => {
   res.status(err.status || 500).send({
```
Comment (AdrianValdes) [inline server.js]: Do we need the `next`?

**Review by AdrianValdes — APPROVED** [review]
Now go in.

**Pull request merged**